x/evm/txs/base/base.go

```diff
@@ -62,7 +62,7 @@ func (tx *Tx) Transition(config types.ChainConfig) (result Result, err error) {
 	}
 
 	// call evm hooks
-	if tmtypes.HigherThanVenus1(tx.Ctx.BlockHeight()) && !tx.Ctx.IsCheckTx() {
+	if tmtypes.HigherThanVenus1(tx.Ctx.BlockHeight()) {
```

Comment:
why delete `&& !tx.Ctx.IsCheckTx()`?

Reply:
when evm call wasm of vmbrdige， user will estimate gas. if disable tx.Ctx.IsCheckTx() ，the gas is worng